routers/api/v1/api.go

```diff
@@ -1,15 +1,7 @@
-// Copyright 2015 The Gogs Authors. All rights reserved.
```

Comment:
Why removing copyrigths ? We could add gitea ones ?

Reply:
Good catch, fixed